### PR TITLE
[SC-6651] update dev framework references -> library

### PR DESCRIPTION
**Pull request by nateshim**
## Internal Notes for Reviewers
Update language from "Developer Framework" to "Library"

Replaced:
"Developer Framework" -> "Library"
"developer framework" -> "library"
"dev framework" -> "library"

## External Release Notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
-# ValidMind Developer Framework
+# ValidMind Library
 
 ![ValidMind logo](images/ValidMind-logo-color.svg "ValidMind logo")
 
-ValidMind's Developer Framework is a library of developer tools and methods designed to run validation tests and automate the documentation of your models. The Developer Framework provides a rich suite of documentation tools and test suites, from documenting descriptions of your dataset to testing your models for weak spots and overfit areas.
+ValidMind's Library is a library of developer tools and methods designed to run validation tests and automate the documentation of your models. The Library provides a rich suite of documentation tools and test suites, from documenting descriptions of your dataset to testing your models for weak spots and overfit areas.
```
Comment (johnwalz97) [inline README.md:5]: bit repetitive 😅

File: README.pypi.md [hunk 2]
```diff
@@ -1,53 +1,53 @@
-# ValidMind Developer Framework
+# ValidMind Library
 
-ValidMind’s Python Developer Framework is a library of developer tools and methods designed to automate
+ValidMind’s Python Library is a library of developer tools and methods designed to automate
```
Comment (johnwalz97) [inline README.pypi.md:3]: same here

File: docs/_build/validmind.html [hunk 3]
```diff
@@ -102,17 +102,17 @@ <h3>Submodules</h3>
     <main class="pdoc">
```
Comment (johnwalz97) [inline docs/_build/validmind.html:102]: anything in 'docs/_build' is automatically generated and commited via github actions so you don't need to manually update :)

**Review by johnwalz97 — APPROVED** [review]
lgtm besides a couple nit picks

File: README.md (hunk 1 above)
Comment (nateshim) [reply README.md]: Changed to 
```
ValidMind's Library includes developer tools and methods designed to run validation tests and automate the documentation of your models. 
```

File: README.pypi.md (hunk 2 above)
Comment (nateshim) [reply README.pypi.md]: Changed to 
```
ValidMind's Library includes developer tools and methods designed to run validation tests and automate the documentation of your models. 
```

File: docs/_build/validmind.html (hunk 3 above)
Comment (nateshim) [reply docs/_build/validmind.html]: removed

**Comment by nateshim** [comment]
@johnwalz97 sorry had to move to new PR because of CLA signing issues: https://github.com/validmind/developer-framework/pull/210

**Pull request closed without merging**